### ISSUE TITLE
Wifi work only on one device

### ISSUE THREAD

**Issue by Eitay** [dukelec/mb#7]
Hello,

I try to connect my Mbrush to my Android Samsung 9 plus device, and even if it manage to connect via WIFI the application can not connect to the printer. 
With ver 4.11 I manage to connect sometimes but not all the time.
On Ver 4,15 it can not connect at all.
Om my old other device that I have LG Android it connect with no problem all the time.
My Sumsung can see the MBrush WIFI and try to connect, but with no sucess.
Please advise if you have any suggestion what should I do to connect the Sumsung, since for now I keep my old LG device just so that I can use the printer with WIFI?

**Comment by dukelec** [comment]
Are LG has no problem with 4.15?

For Samsung, the WiFi connection was established, but APP can't load, please refer to section `WiFi connection issues` in [http://blog.dukelec.com/mb-faq](http://blog.dukelec.com/mb-faq)

But, I think the issue is likely as same as #6, you can take the same action I relied in #6.

**Comment by Eitay** [comment]
No problem with the LG on both version 4.11 and 4.15 that I upgrade today.
On the 4.11 I could connect only sometimes, with the 4.15 I manage to connect 2 times when I modify the phone IP to 192.168.44.3 , but the APP could not connect.
As I said, on 4.11 there where times it work and times it wouldnt work, that is why I change to the LG that work every time.
Do you think it is still a hardware problem even when it work with the LG phone?
Thank you

**Comment by dukelec** [comment]
> Do you think it is still a hardware problem even when it work with the LG phone?

Yes, different device have different frequency deviation (even for same model), for example:
MBrush: +30ppm
LG: +20ppm
Samsung: -20ppm
The deviation between MBrush and LG is 10ppm, but for MBrush and Samsung is 50ppm.

**Comment by Eitay** [comment]
So if I got it from Geekbuying, i have to contact them for the solustion, am I right?

**Comment by dukelec** [comment]
Yes, you can send this issue link to them.
I'm sorry for this issue.

**Comment by Eitay** [comment]
OK, thank you for the help
